### ISSUE TITLE
JVM crash when `VideoCapture.release()`

### ISSUE THREAD

**Issue by penguin410** [bytedeco/javacv#781]
I encountered following error when attempted `VideoCapture.release()`.

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fff8bfce49f, pid=73979, tid=0x0000000000000307
#
# JRE version: Java(TM) SE Runtime Environment (8.0_111-b14) (build 1.8.0_111-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.111-b14 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# C  [libobjc.A.dylib+0x949f]  objc_release+0x1f
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
```

my code:
```
import org.bytedeco.javacpp.opencv_videoio.*;

public class VideoCaptureTest {
    public static void main(String[] args) {
        VideoCapture capture = new VideoCapture(0);
        capture.release();
    }
}
```

Environment
* macOS Sierra 10.12.6
* JRE build 1.8.0_111-b14
* org.bytedeco:javacv-platform:1.3.3

Is there a solution?

**Comment by saudet** [comment]
FFmpeg usually supports video capture better than OpenCV, so you might want to try that with FFmpegFrameGrabber and the appropriate device name and format:
https://trac.ffmpeg.org/wiki/Capture/Webcam#OSX

**Comment by saudet** [comment]
This appears to be fixed in OpenCV 3.3.0: https://github.com/opencv/opencv/issues/7833

**Comment by penguin410** [comment]
Thank you for the information!

But, I could not find a way to use the FFMpegFrameGrabber to control the camera from JavaCV.
For example, it is the following code.
`FFMpegFrameGrabber grabber = new FFMpegFrameGrabber(0); // '0' is device number.`

For some reason I can't execute ffmpeg on the command line.
(I only have permission to use JavaCV.) 


> This appears to be fixed in OpenCV 3.3.0: opencv/opencv#7833

That is very helpful information.
I'll try using OpenCV3.3.0 when will JavaCV be able to support OpenCV3.3.0.

**Comment by saudet** [comment]
The newly released version 1.4 is based on OpenCV 3.4.0, so this should be fixed now.
